spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/config/SqsEndpoint.java

```diff
@@ -88,6 +91,15 @@ public Integer getMaxMessagesPerPoll() {
 		return this.maxMessagesPerPoll;
 	}
 
+	/**
+	 * Return whether message should be grouped or not.
+	 * @return the message grouping.
+	 */
+	@Nullable
+	public Boolean getMessageGrouping() {
```

Comment:
How about instead of having a boolean for this configuration, which can be a bit cryptic sometimes and is hard to evolve, we use an enum instead?

Something like:
`FifoBatchGroupingStrategy` 
PROCESS_MESSAGE_GROUPS_IN_PARALLEL
RECEIVE_ALL_GROUPS_IN_SAME_BATCH

Not sold on the names, but something similar might give users a better sense of what the options are, and allows us to add more possibilities down the road should the need appear.

What do you think?

Reply:
I did not think about it, but I think it's a great idea, thank you!